src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/Management/ManagementClientTests.cs

```diff
@@ -27,61 +28,69 @@ public async Task BasicQueueCrudTest()
-
-            await client.DeleteQueueAsync(updatedQ.Path);
-
-            await Assert.ThrowsAsync<MessagingEntityNotFoundException>(
-                    async () =>
-                    {
-                        await client.GetQueueAsync(qd.Path);
-                    });
-
-            exists = await client.QueueExistsAsync(queueName);
-            Assert.False(exists);
+                var qd = new QueueDescription(queueName)
+                {
+                    AutoDeleteOnIdle = TimeSpan.FromHours(1),
+                    DefaultMessageTimeToLive = TimeSpan.FromDays(2),
+                    DuplicateDetectionHistoryTimeWindow = TimeSpan.FromMinutes(1),
+                    EnableBatchedOperations = true,
+                    EnableDeadLetteringOnMessageExpiration = true,
+                    EnablePartitioning = false,
+                    ForwardDeadLetteredMessagesTo = null,
+                    ForwardTo = null,
+                    LockDuration = TimeSpan.FromSeconds(45),
+                    MaxDeliveryCount = 8,
+                    MaxSizeInMB = 2048,
+                    RequiresDuplicateDetection = true,
+                    RequiresSession = true,
+                    UserMetadata = nameof(BasicQueueCrudTest)
+                };
+
+                qd.AuthorizationRules.Add(new SharedAccessAuthorizationRule(
+                    "allClaims",
+                    new[] { AccessRights.Manage, AccessRights.Send, AccessRights.Listen }));
+
+                var finalQ = await client.CreateQueueAsync(qd);
+                Assert.Equal(qd, finalQ);
+
+                var getQ = await client.GetQueueAsync(qd.Path);
+                Assert.Equal(qd, getQ);
+
+                getQ.EnableBatchedOperations = false;
+                getQ.MaxDeliveryCount = 9;
+                getQ.AuthorizationRules.Clear();
+                getQ.AuthorizationRules.Add(new SharedAccessAuthorizationRule(
+                    "noManage",
+                    new[] { AccessRights.Send, AccessRights.Listen }));
+
+                var updatedQ = await client.UpdateQueueAsync(getQ);
+                Assert.Equal(getQ, updatedQ);
+
+                var exists = await client.QueueExistsAsync(queueName);
+                Assert.True(exists);
+
+                var queues = await client.GetQueuesAsync();
+                Assert.True(queues.Count >= 1);
+                Assert.Contains(queues, e => e.Path.Equals(queueName, StringComparison.OrdinalIgnoreCase));
+
+                await client.DeleteQueueAsync(updatedQ.Path);
+
+                await Assert.ThrowsAsync<MessagingEntityNotFoundException>(
+                        async () =>
+                        {
+                            await client.GetQueueAsync(qd.Path);
+                        });
+
+                exists = await client.QueueExistsAsync(queueName);
+                Assert.False(exists);
+            }
+            catch
```

Comment:
These tests are not running in the scope of a given Service Bus resource, they're testing the operations for creating and managing them.   The lambda scope elsewhere is responsible for setting up the resource that the test expects so that it runs in the scope.... I gave a look do doing something similar here, but it seemed over complicated, so I backed it out and went simple try/catch.

With respect to `catch` over `finally`, the tests do their own cleanup and then assert against that in the normal flow, so unless there's a failure or unexpected exception, the additional code that I added shouldn't have to run.    But, these have left orphaned resources in the Service Bus namespace when they did have a failure/exception, so I added an additional layer of cleanup for that corner case.